apps/api/src/app/events/usecases/send-message/send-message-in-app.usecase.ts

```diff
@@ -46,9 +47,16 @@ export class SendMessageInApp extends SendMessageBase {
     protected createExecutionDetails: CreateExecutionDetails,
     protected subscriberRepository: SubscriberRepository,
     private compileTemplate: CompileTemplate,
-    private organizationRepository: OrganizationRepository
+    private organizationRepository: OrganizationRepository,
+    protected getDecryptedIntegrationsUsecase: GetDecryptedIntegrations
   ) {
-    super(messageRepository, createLogUsecase, createExecutionDetails, subscriberRepository);
+    super(
+      messageRepository,
+      createLogUsecase,
+      createExecutionDetails,
+      subscriberRepository,
+      getDecryptedIntegrationsUsecase
+    );
```

Comment:
added `GetDecryptedIntegrations` as it's required by the super class, check below